modules/ROOT/pages/install-prereqs.adoc

```diff
@@ -4,12 +4,32 @@ include::_attributes.adoc[]
 endif::[]
 :page-aliases: install-port-reqs.adoc, install-sys-reqs.adoc
 
-Review the following requirements with your operations and IT teams to ensure that all prerequisites are met before installing Anypoint Runtime Fabric. The Runtime Fabric installation process fails if the minimum requirements are not met.
+== Before You Begin
+
+[IMPORTANT]
+Most Anypoint Runtime Fabric issues result from improper infrastructure provisioning or configuration. 
+
+Before you install Runtime Fabric :
+
+* Refer to xref:index.adoc#shared-responsibility-for-on-premises-runtime-fabric[Shared Responsibility for On-Premises Runtime Fabric].
++
+If your infrastructure does not meet the minimum hardware, operating system, and networking requirements, Runtime Fabric will not operate successfully.
+
+* Enlist the assistance of your organization’s:
+** DevOps team to provision and manage the infrastructure
+** Network team to whitelist ports and configure proxy settings
+** Security team to review for compliance and obtain security certificates
++
+Working with these teams in your organization helps ensure a successful installation and avoids performance and density issues. 
+
+* Review the following requirements and ensure that all prerequisites are met before installing Anypoint Runtime Fabric.
```

Comment:
Which "following requirements"?
Do you mean subsequent subtopics?
If so, avoid using language in one topic that refers to another, in case topics are reused or the audience is not reading everything chronologically.
I suggest something more like...:
```suggestion
* Ensure that you have the general knowledge required to perform and installation [and whatever topics follow that comprise "the following requirements"].
```